File: 프로그래머스/lv0/120839. 가위 바위 보/가위 바위 보.cs
using System;
using System.Text;

public class Solution {
    public string solution(string rsp) {
        
        StringBuilder sb = new StringBuilder(rsp.Length);
        
        char[] charArray = rsp.ToCharArray();
        foreach(char item in charArray)
        {
            char currnt = GetWinStr(item);
            sb.Append(currnt);
        }
        
        return sb.ToString();
    }
    
    public char GetWinStr(char n)
    {
        switch(n)
        {
            case '2': return '0';
            case '0': return '5';
            case '5': return '2';
            default : return default(char);
        }
    }
}

File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.cpp
#include <string>
#include <vector>
#include <map>

using namespace std;

string solution(string rsp) {
    string answer = "";
    
    map<char, char> map = { {'2', '0'}, {'0', '5'}, {'5', '2'} };
    
    for (const char& c : rsp)
    {
        answer += map[c];
    }
    
    return answer;
}